tests/v2/validation/terratest/config.go

```diff
@@ -0,0 +1,78 @@
+package terratest
+
+type GoogleAuthEncodedJSON struct {
+	AuthProviderX509CertURL string `json:"auth_provider_x509_cert_url" yaml:"auth_provider_x509_cert_url"`
+	AuthURI                 string `json:"auth_uri" yaml:"auth_uri"`
+	ClientEmail             string `json:"client_email" yaml:"client_email"`
+	ClientID                string `json:"client_id" yaml:"client_id"`
+	ClientX509CertURL       string `json:"client_x509_cert_url" yaml:"client_x509_cert_url"`
+	PrivateKey              string `json:"private_key" yaml:"private_key"`
+	PrivateKeyID            string `json:"private_key_id" yaml:"private_key_id"`
+	ProjectID               string `json:"project_id" yaml:"project_id"`
+	TokenURI                string `json:"token_uri" yaml:"token_uri"`
+	Type                    string `json:"type" yaml:"type"`
+}
+
+type Nodepool struct {
+	Quantity         int64  `json:"quantity" yaml:"quantity"`
+	Etcd             bool   `json:"etcd" yaml:"etcd"`
+	Cp               bool   `json:"cp" yaml:"cp"`
+	Wkr              bool   `json:"wkr" yaml:"wkr"`
+	InstanceType     string `json:"instanceType" yaml:"instanceType"`
+	DesiredSize      int64  `json:"desiredSize" yaml:"desiredSize"`
+	MaxSize          int64  `json:"maxSize" yaml:"maxSize"`
+	MinSize          int64  `json:"minSize" yaml:"minSize"`
+	MaxPodsContraint int64  `json:"maxPodsContraint" yaml:"maxPodsContraint"`
+}
+
+type TerraformConfig struct {
+	Ami                                 string   `json:"ami" yaml:"ami"`
```

Comment:
[clarification]:
I noticed in the example jobs this is set to be blank. Is there a default set somewhere? If not, why does this work with a blank ami? 
i.e. 
```
terraform:
  ami: 
  region: us-east-2
...
```

Reply:
This field isn't required. so by default should be left blank. (terraform will use default if left blank).  It may be specified though, so for specific use cases, it is important to have this field as an option available in the config.

Would this be a situation to use `omitempty`?  If so, i can update this, so `ami` field, may be left out of config unless specifically needed to be defined for the test case

Reply:
yes that's correct. `omitempty` is what to use in `go` for optional variables.